GitUpKit/Interface/GIGraphView.m

```diff
@@ -860,97 +860,168 @@ static void _DrawLine(GILine* line, CGContextRef context, CGFloat offset, CGFloa
   }
 }
 
-static void _DrawBranchTitle(CGContextRef context, CGFloat x, CGFloat y, GIBranch* branch, NSDictionary* attributes, GIGraphOptions options) {
-  NSUInteger index;
+static void _DrawBranchTitle(CGContextRef context, CGFloat x, CGFloat y, NSColor* color, GIBranch* branch, GIGraphOptions options) {
   
-  // Generate text
+  // Build a long rich text from branches and tags
+  
```

Comment:
There was a blank line before I started making changes, so I mistakenly decided it is a part of code style 😊 Fixed.